Dockerfile

```diff
@@ -10,26 +10,27 @@ WORKDIR /stk
 ENV VERSION=1.1
 
 # Install build dependencies
-ENV DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-    apt-get install -y build-essential \
+    apt-get install --no-install-recommends -y build-essential \
                        cmake \
                        git \
                        libcurl4-openssl-dev \
                        libenet-dev \
                        libssl-dev \
                        pkg-config \
                        subversion \
-                       zlib1g-dev
+                       zlib1g-dev \
+                       ca-certificates
 
 # Get code and assets
 RUN git clone --branch ${VERSION} --depth=1 https://github.com/supertuxkart/stk-code.git
-RUN svn checkout https://svn.code.sf.net/p/supertuxkart/code/stk-assets-release/${VERSION}/ stk-assets
+RUN svn checkout https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
 
 # Build server
 RUN mkdir stk-code/cmake_build && \
     cd stk-code/cmake_build && \
-    cmake .. -DSERVER_ONLY=ON && \
+    cmake .. -DSERVER_ONLY=ON -USE_SYSTEM_ENET=ON && \
```

Comment:
Could you explain the benefit of this option?

Reply:
It instructs cmake to use the systems enet instead of the embedded one which is used in some other dockerfiles as well. Generally a good idea to use libraries provided by the system. (http://enet.bespin.org/)

Reply:
Seems reasonable. :+1: